### 2x2 Contingency Table

Prospective Randomized study 

![table1](table1.png)

#### Sample Odds Ratio 

$$\hat{\Theta} = \frac{n_{11}n_{22}}{n_{12}n_{21}} = \frac{(1)(379)}{(377)(1)} = 1.00531$$

##### Interpretation

The estimated odds of AE occuring (사례발생) for someone taking 온베브지 is 1.00531 times more likely of AE occuring for someone taking 아바스틴. Since $\hat{\Theta} \approx 1$, there is no association between exposure group (drug) vs. outcome (AE occuring).

#### Hypothesis Testing: Likelihood Ratio Test

If $\hat{\Theta} \gg 1$, then perform hypothesis test for statistical significance. 

데이터 및 변수 입력 cell:

In [8]:
n_11 = 1
n_12 = 377
n_21 = 1
n_22 = 379

data = matrix(c(n_11, n_21, n_12, n_22), nrow = 2, ncol = 2)
colnames(data) = c("사례 발생", "사례 미발생")
rownames(data) = c("온베브지", "아바스틴")

data

사례 발생 사례 미발생
온베브지 1         377        
아바스틴 1         379

Liklihood ratio test cell:

For 95% significance, compare `m1$null.deviance` with `qchisq(0.95, 1)`

In [10]:
m1 = glm(data ~ c(1, 0), family = binomial)
m1$null.deviance

[1] 1.396048e-05

In [11]:
qchisq(0.95, 1)

[1] 3.841459

* If `m1$null.deviance` > `qchisq(0.95, 1)`, conclude that exposure (drug) and outcome (AE occuring) is **not** independent (reject $H_{0}$)
* If `m1$null.deviance` < `qchisq(0.95, 1)`, conclude that exposure (drug) and outcome (AE occuring) is independent (reject $H_{0}$)

Example:
$1.396 \times 10^{-5} \approx 0 \lt 3.841 \implies$ exposure (drug) and outcome (AE occuring) is independent (i.e., no association)